XXX-icu-module.md

```diff
@@ -0,0 +1,137 @@
+| Title  | ICU Module                  |
+|--------|-----------------------------|
+| Author | @jasnell                    |
+| Status | REJECTED                    |
```

Comment:
Ha!! Copied the header from a different one lol
On Apr 27, 2016 2:56 PM, "Trevor Norris" <notifications@github.com> wrote:

> In XXX-icu-module.md
> <https://github.com/nodejs/node-eps/pull/15#discussion_r61343698>:
>
> > @@ -0,0 +1,137 @@
> > +| Title  | ICU Module                  |
> > +|--------|-----------------------------|
> > +| Author | @jasnell                    |
> > +| Status | REJECTED                    |
>
> Just curious. Why is this labeled as REJECTED?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node-eps/pull/15/files/3fad3f260a0533ef0fead0d640ac2c3cc78d2d0a#r61343698>
>